projects/demo/src/modules/app/stackblitz/project-files/configs.md

```diff
@@ -3,54 +3,59 @@
 ```json
```

Comment:
`angular.json` & `tsconfig.json` are copy-pasted from the most recent angular stackblitz starter (angular 18+):

* https://github.com/stackblitz/starters/blob/main/angular/angular.json
* https://github.com/stackblitz/starters/blob/main/angular/tsconfig.json